src/loader.ts

```diff
@@ -25,23 +26,19 @@ module.exports = function (source: any) {
 
     let codeTransformed = source
 
-    const cb = this.async()
+    // Scoped styles have paths like this: "foo.scoped.css?scoped=true"
+    const scopedStyles = resourcePath.endsWith('.css') &&
+        resourceQuery &&
+        new URLSearchParams(resourceQuery).get('scoped') === 'true'
 
-    compiler
-        .transform(codeTransformed, resourcePath, {
+    const { code } = compiler.transformSync(codeTransformed, resourcePath, {
             name: info.name,
             namespace: info.ns,
             stylesheetConfig,
             outputConfig,
             experimentalDynamicComponent,
-            scopedStyles: resourcePath.endsWith('.scoped.css')
-        })
-        .then((res: any) => {
-            cb(null, res.code)
-        })
-        .catch((err: any) => {
-            cb(err)
+            scopedStyles
         })
```

Comment:
I figured I may as well refactor this to use `transformSync` rather than the deprecated `transform`. `transformSync` has [been available](https://unpkg.com/browse/@lwc/compiler@2.0.0/dist/commonjs/index.js) since v2.0.0 of `@lwc/compiler`, and we have our peerDep set to `^2.0.0`.